### ISSUE TITLE
Error starting Jetty

### ISSUE THREAD

**Issue by rkrawiec** [camelinaction/camelinaction2#28]
When i do a mvn spring-boot:run I get the exception below. It appears Jetty wants port 8080 but Tomcat already has it. Full Stacktrace attached.
[stacktrace.txt](https://github.com/camelinaction/camelinaction2/files/769469/stacktrace.txt)

org.apache.camel.spring.boot.CamelSpringBootInitializationException: org.apache.cxf.interceptor.Fault: Could not start Jetty server on port 8,080: Address already in use: bind

Edit: Excluding Tomcat and adding a Jetty dependency results in the same error except Jetty already has the port.

**Comment by davsclaus** [comment]
Maybe you have another JVM running that has port 8080, as this example works fine for me

**Comment by rkrawiec** [comment]
I did a netstat -a -n -o | find "8080" before I started the app and there was no output.

I found that if i changed the port in the application.yml to 8090 it will start. but i get two jetty servers one on 8080 and one on 8090 which is what I changed the port to in the .yml. If i change the port in the cxfrs route to 8090 it fails again. Its seems CXF is starting its own Jetty instance rather than using the existing one?

There were clearly two Jetty servers starting in the dump i attached previously, im just not sure why.

below is the netstat output of both ports after starting the application.
`C:\Users\Rob>netstat -a -n -o | find "8080"
  TCP    127.0.0.1:8080         0.0.0.0:0              LISTENING       4696

C:\Users\Rob>netstat -a -n -o | find "8090"
  TCP    0.0.0.0:8090           0.0.0.0:0              LISTENING       4696
  TCP    [::]:8090              [::]:0                 LISTENING       4696

C:\Users\Rob>`
